docs/development.md

```diff
@@ -96,3 +97,15 @@
     ```sh
     wrk -t2 -c2 -d30s -s post.lua http://<api-service-address>/api/v1/projects/<my-project>/analyze
     ```
+
+
+## Running in kubernetes
+
+    1. If deploying from a private registry, verify that Kubernetes has access to the [Docker Registry](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-auth-aks).
```

Comment:
mention that this link is an example relevant just for AKS not any k8s

Reply:
The link shows also k8s native ways to do it, using secrets. it's not aks specific